docs/tutorials/get-tasks-by-title

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+---
+
+# Tutorial: Get tasks by title
+
+In this tutorial, you learn the operations to return a list of tasks with the same title.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+Make sure you've completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you'll use for the tutorial.
+
+## Get tasks by title
+
+Returning a list of tasks from the service requires you to perform a get (`GET`) request on the [`task`](../api/task) resources in the service.
+
+To get tasks by their title:
+
+1. Make sure your local service is running. If it is not running, start it by using this command:
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+1. Open the Postman app on your desktop.
+1. In the Postman app, create a new request with these values:
+    * **METHOD**: GET
+    * **URL**: `{{base_url}}/tasks/title`
+    * **Headers**:
+        * `Content-Type: application/json`
+
+1. In the Postman app, select **Send** to make the request.
+1. Watch for the response body, which should look something like this:
+
+    ```js
+    {
+        "user_id": 3,
+        "title": "Get new tires",
+        "description": "Get new tires for Hoppity",
+        "due_date": "2024-03-11T14:00",
+        "warning": "-60",
+        "id": 5
+    }
+    ```
```

Comment:
Your tutorial should have a **Conclusion** and/or **Next steps** section so the reader knows that they've completed the task and get some ideas about what else they could do.